hydra-node/src/Hydra/Chain/Direct/ScriptRegistry.hs

```diff
@@ -156,13 +158,15 @@ registryUTxO scriptRegistry =
 --
 -- Can throw at least 'NewScriptRegistryException' on failure.
 queryScriptRegistry ::
-  (MonadIO m, MonadThrow m) =>
+  -- | Current network discriminant
```

Comment:
Nitpick: "Current" does not make sense here. This will never change for a given network. Also, the network discriminant is strictly speaking `Mainnet | Testnet`, where the the network id is a combination of discriminant + magic number.